tests/switchless/host/host.c

```diff
@@ -62,26 +64,38 @@ int main(int argc, const char* argv[])
              argv[1], OE_ENCLAVE_TYPE_SGX, flags, NULL, 0, &enclave)) != OE_OK)
         oe_put_err("oe_create_enclave(): result=%u", result);
 
-    pthread_t threads[NUM_HOST_THREADS];
-    for (int i = 0; i < NUM_HOST_THREADS; i++)
-    {
-        int ret = 0;
-        if ((ret = pthread_create(&threads[i], 0, host_thread, enclave)))
-        {
-            oe_put_err("pthread_create(host): ret=%u", ret);
-        }
-        else
-            printf("created thread %u\n", i);
-    }
+    oe_start_switchless_manager(enclave, 2);
 
-    for (int i = 0; i < NUM_HOST_THREADS; i++)
-    {
-        pthread_join(threads[i], NULL);
-    }
+    char out[100];
+    int return_val;
+
+    double switchless_microseconds = 0;
+    struct timespec start, end;
+
+    clock_gettime(CLOCK_REALTIME, &start);
```

Comment:
A million iteration takes 14 seconds in this case (1 for switchless calls, 13 for regular calls). I think it's OK considering many other test cases run significantly longer.

When we implement #2129, it would be a good idea to parameterize it.

Reply:
Think of it this way, you're taxing every developer an extra 14 seconds when they run regression tests and that really adds up. Of the >100 tests only the crypto and thread tests take >10s to run and those really should be fixed. Just because we already have excessively long running tests doesn't mean we should add more of them. If we are not implementing #2129 until later, then don't put in the extra iterations right now, just take them out completely.

Reply:
Make sense. Will do.